indico/modules/rb_new/client/js/actions.js

```diff
@@ -462,3 +474,43 @@ export function fetchBookingAvailability(room, filters) {
         ({availability, date_range: dateRange}) => ({availability: availability[room.id], dateRange})
     );
 }
+
+export function fetchBlockingsStarted() {
```

Comment:
let's not add action creators for "internal" actions that are only dispatched from within another action. if there's an argument like a namespace it kind of makes sense (but even then i wouldn't *export* it) - but for something as simple as this one, just `dispatch({type: ...})` directly